spec/views/partials/multi_factor_authentication/_mfa_selection.html.erb_spec.rb

```diff
@@ -39,7 +39,7 @@
     subject(:rendered) do
       render partial: 'mfa_selection', locals: {
         form: form_builder,
-        option: presenter.options[3],
+        option: presenter.options.find { |option| option.disabled? },
```

Comment:
I'm a bit confused by what this option is/was doing. What is the option for? If the context is about an existing configuration and we're setting up a TOTP authentication app, maybe we could look for the option specifically for the TOTP app (by name/class)?

Reply:
Yea so I changed this mainly because before we were just looking for the certain element in the array. I could also search for TOTP configuration as well.